src/Spectre.libGenetic/MinimalLengthCondition.h

```diff
@@ -0,0 +1,49 @@
+/*
+* MinimalLengthCondition.h
+* Checks, if Individual is at least of specific length.
+*
+Copyright 2018 Spectre Team
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+#include "Spectre.libGenetic/BaseIndividualFeasibilityCondition.h"
+
+namespace spectre::algorithm::genetic
+{
+/// <summary>
+/// Checks, if Individual is at least of specific length.
+/// </summary>
+class MinimalLengthCondition : public BaseIndividualFeasibilityCondition
```

Comment:
If we have this, we should also go for `MaximalLengthCondition`. Do we need this condition by the way? The only validation against `Individual` length is *exactly* the same length as *something* (training set in our case). Maybe we can redefine it to specify only the single admissible `Individual` length?

Reply:
done